modules/system/assets/ui/js/tab.js

```diff
@@ -33,10 +33,22 @@
             self.initTab(this)
         })
 
-        this.$el.on('close.oc.tab', function(ev, data){
-            ev.preventDefault()
-            var force = (data !== undefined && data.force !== undefined) ? data.force : false;
-            self.closeTab($(ev.target).closest('ul.nav-tabs > li, div.tab-content > div'), force)
+        this.$el.on('close.oc.tab', function (ev, data) {
+            if (2 == ev.which) {
```

Comment:
I see no difference in the branches here, they both run the same code whether or not `2 == ev.which`

Reply:
Make the separate function that looks roughly like this:
```js
this.$el.on('click', function (ev) {
    if (ev.which === 2) {
        $(ev.target).trigger('close.oc.tab');
    }
});
```

Reply:
Done!